src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -211,6 +208,11 @@ private void validateSchema(final EventType eventType) throws InvalidEventTypeEx
                 throw new InvalidEventTypeException("\"metadata\" property is reserved");
             }
 
+            if (eventType.getCompatibilityMode() == CompatibilityMode.DEPRECATED) {
+                throw new InvalidEventTypeException(
+                        "\"compatibility_mode\" should be either \"compatible\" or \"none\"");
```

Comment:
@v-stepanov correct. Also, deprecated schemas should not be upgradable to compatible neither become "none". At least in this PR. There is another ticket to allow for upgrading a deprecated schema into one of the other two options. It will be done next and will require some additional checks.